code/core/scripts/helpers/generateTypesMapperFiles.ts

```diff
@@ -12,7 +12,7 @@ async function generateTypesMapperContent(filePath: string) {
   const downwards = relative(cwd, filePath);
 
   return dedent`
-    // auto generated file from ${__filename}, do not edit
+    // auto generated file from ${import.meta.url}, do not edit
```

Comment:
We should likely make this a relative path; or just hard-code it...

Reply:
yeah it should probably be relative.